internal/operators/manager.go

```diff
@@ -105,31 +108,50 @@ func (mgr *Manager) GenerateManifests(ctx context.Context, cluster *common.Clust
 
 		operator := mgr.olmOperators[clusterOperator.Name]
 		if operator != nil {
-			manifests, err := operator.GenerateManifests(cluster)
+			openshiftManifests, manifests, err := operator.GenerateManifests(cluster)
 			if err != nil {
 				mgr.log.Error(fmt.Sprintf("Cannot generate %s manifests due to ", clusterOperator.Name), err)
 				return err
 			}
-			for k, v := range manifests {
-				err = mgr.createManifests(ctx, cluster, k, v)
+			for k, v := range openshiftManifests {
+				err = mgr.createManifests(ctx, cluster, k, v, models.ManifestFolderOpenshift)
 				if err != nil {
 					return err
 				}
 			}
+
+			for _, v := range manifests {
+				customManifest = fmt.Sprintf("%s---\n%s\n", customManifest, v)
+			}
+
+		}
+	}
+
+	if customManifest != "" {
+		if err := mgr.createCustomManifest(ctx, cluster, customManifest); err != nil {
+			return err
 		}
 	}
 
 	return nil
 }
 
-func (mgr *Manager) createManifests(ctx context.Context, cluster *common.Cluster, filename string, content []byte) error {
+func (mgr *Manager) createCustomManifest(ctx context.Context, cluster *common.Cluster, content string) error {
```

Comment:
Consider adding a comment explaining that this is for custom manifests that requires CRDs are created late by the cluster operators (and not by the CVO)